common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -31,7 +31,10 @@ export enum MessageType {
     // Channel operation.
     Operation = "op",
 
-    // Forced snapshot
+    /**
+     * Forced snapshot
+     * @deprecated Do not use save ops - See issue #8288
```

Comment:
Why not delete it right away?
Even if someone is using it, there is an easy work around - use "saveOp" or define your own constant.

Reply:
I just thought it is standard practice to deprecate before removing. Is it okay to remove these APIs without deprecation because they are unused in this situation?

Reply:
I'd say that the core goal is for consumers not to be forced to do expensive work (including changes in logic / behavior) in branches where they consume FF releases. That's the primary reason to deprecate things and let them deal with it in main / separate branches before something is gone. At least when it comes to API changes (not other types of changes, like document schema changes).

Also, in places where we promise back-compat, we quite often need to leave API in place, even if they are deprecated / removed from interfaces (as previously compiled code needs to continue to work with newer FF code). There, occasionally, it also makes sense to remove (not deprecate) things from interfaces (not implementation!) if there is an obvious substation, i.e., force (small) action sooner.

It's a balance between your time (more PRs) + forced action and convenience / not be forced to do expensive work right away. Given that definition packages are consumed with delay, and runtime behavior is removed (essentially sooner), there is really not much value in keeping it here.